lib/galaxy/tool_util/verify/asserts/__init__.py

```diff
@@ -12,16 +12,19 @@
 # create a new module of assertion functions, create the needed python
 # source file "test/base/asserts/<MODULE_NAME>.py" and add
 # <MODULE_NAME> to the list of assertion module names defined above.
-assertion_modules = []
+assertion_functions = {}
```

Comment:
@simonbray that's going to affect https://github.com/galaxyproject/planemo/pull/1213/, I guess. This should be easier to use down the line though.